### PR TITLE
Update to dbt version 1.0

### PR DESCRIPTION
**Pull request by epapineau**
## Description & motivation

Update Rapid Onboarding Exemplar project to use dbt 1.0:

- Update package versions
- Require dbt version 1.0 or greater

## Changes to existing models:

No changes to existing models, but now explicitly materializing models in the intermediate directory as views.

## Validation of models:

Screen Shot 2022-03-14 at 9 04 49 PM
Screen Shot 2022-03-14 at 9 04 13 PM

## Checklist:

- [x] My pull request represents one logical piece of work.
- [x] My commits are related to the pull request and look clean.
- [x] My SQL follows the [dbt Labs style guide](https://github.com/dbt-labs/corp/blob/master/dbt_style_guide.md).
- [x] I have materialized my models appropriately.

### CODE REVIEW

**Review by patkearns10 — APPROVED** [review]
LGTM!
🌵 
What is the v3.json vs v4.json issue you mentioned in Slack?

File: README.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
-# {{ organization_name }} Analytics
+# dbt Training - Rapid Onboarding Exemplar
```
Comment (christineberger) [inline README.md:1]: Any reason why this template changed? This is the template for the RO and we have the client copy this / add it to their own repo and modify it

**Comment by epapineau** [comment]
> LGTM! 🌵 What is the v3.json vs v4.json issue you mentioned in Slack?

@patkearns10 The CI/CD job uses `manifest.json` to identify where `state:modified`. However, between dbt 0.21 and dbt 1.0 `manifest.json` changed from version: 3 to version: 4 and are not comparable. This results in a CI/CD failure when the last successful production job is on a dbt 0.21, as observed [here](https://cloud.getdbt.com/#/accounts/26712/projects/48031/runs/47783786/).

File: README.md (hunk 1 above)
Comment (epapineau) [reply README.md:1]: I was updating to have an example of what a populated template would look like. Happy to revert if there's a strong opinion for keeping it the template, but the RO lesson plan [links to your gist](https://gist.github.com/christineberger/2d00a2f846b1b49359fde5c2f8f354fe), not the example project repo.

Comment (patkearns10) [reply README.md:1]: I think it is confusing to both be an example and be a template for building your own.
I would opt for a filled out README and separately a template, either in this repo or another repo.

Comment (graciegoheen) [reply README.md:1]: +1 To Pat's idea

File: dbt_project.yml [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 # These configurations specify where dbt should look for different types of files.
-# The `source-paths` config, for example, states that models in this project can be
+# The `model-paths` config, for example, states that models in this project can be
 # found in the "models/" directory. You probably won't need to change these!
-source-paths: ["models"]
+model-paths: ["models"]
 analysis-paths: ["analysis"]
 test-paths: ["tests"]
-data-paths: ["data"]
+seed-paths: ["data"]
```
Comment (graciegoheen) [inline dbt_project.yml:18]: Should we change this folder to be named "seeds" to be aligned with our [seeds docs](https://docs.getdbt.com/docs/building-a-dbt-project/seeds)?

**Review by graciegoheen — CHANGES_REQUESTED** [review]
Looks good! Just one adjustment on re-naming the seeds directory!

File: dbt_project.yml (hunk 2 above)
Comment (epapineau) [reply dbt_project.yml]: Good eyes @graciegoheen ! Just updated

**Pull request merged**